jenkins/release-workflows/release-chores.jenkinsfile

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+lib = library(identifier: 'jenkins@refactor-classes', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/gaiksaya/opensearch-build-libraries.git',
+]))
```

Comment:
To be changed once https://github.com/opensearch-project/opensearch-build-libraries/pull/582 is merged and `8.2.0` tag is cut. Same goes for test file as well.